node_helper.js

```diff
@@ -89,6 +89,13 @@ module.exports = NodeHelper.create({
           "user-agent": "MagicMirror:MMM-Wallpaper:v1.0 (by /u/kolbyhack)"
         },
       });
+      else if (source.startsWith("/user/")) {
```

Comment:
You should be able to avoid this extra block by just changing line 85 to 
`} else if (source.startsWith("/r/") || source.startsWith("/user/") {`
can you confirm and update?

Reply:
Unfortunately, I noticed an error in my original code. User's multireddit don't use /hot, or at least the url doesn't need it. Combining the two actually breaks the multireddit section. I originally hard coded it test if multireddits were possible, but didn't notice the error till after. 

I have updated the file to have both /r/ and /users/ as separate. But combined where they are called later on line 178 since that functionality works for both.